packages/components/src/components/modal/modal.scss

```diff
@@ -86,6 +86,10 @@
         @include desktop {
             min-width: 440px !important;
             max-height: calc(100vh - #{$HEADER_HEIGHT} - #{$FOOTER_HEIGHT}) !important;
+
+            &_jurisdiction-modal {
+                max-height: none !important;
+            }
```

Comment:
why do we add this style in modal.scss?
we already have below styling in cfd.scss
.dc-modal__container_jurisdiction-modal {
    .dc-modal-header {
        border-bottom: 2px solid var(--general-section-1);
    }
}
cant we add it here instead?

Reply:
Will try this approach and check whether it works 🙏